tests/Unit/RunSingleTest/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(EXECUTABLE "RunSingleTest")
+
+add_executable(
+  ${EXECUTABLE}
+  EXCLUDE_FROM_ALL
+  ${EXECUTABLE}.cpp
+  ../DataStructures/Test_DataVector.cpp
```

Comment:
This example would be clearer to new users if you add a comment above this line, saying that the following lines list the test (or tests, if multiple tests are given) that RunSingleTest can run.